### PR TITLE
Add WBTC keeper support

### PR DESCRIPTION
**Pull request by grandizzy**
- this needs auction keeper WBTC support to be committed
- existing keepers will have to `./cleanup.sh` in order to rebuild Docker image with WBTC support

### CODE REVIEW

File: flip/wbtc-a/model-wbtc.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+while true; do
+
+  # share FLIP_WBTC_URL, FLIP_WBTC_DISCOUNT, and GASPRICE_MULTIPLIER
+  source ../env/environment.sh
+
+  # dynamic bid price
+  body=$(curl -s -X GET "${FLIP_WBTC_URL}" -H "accept: application/json")
+  bitcoinPrice=$(echo ${body} | jq '.bitcoin.usd')
+  bidPrice=$(bc -l <<< "${bitcoinPrice} * (1-${FLIP_WBTC_DISCOUNT})")
```
Comment (EdNoepel) [inline flip/wbtc-a/model-wbtc.sh:14]: Had to look this up; I've never used `bc`.  Does it have advantages over `bash` arithmetic?

**Review by EdNoepel — APPROVED** [review]
Note that existing `auction-keeper` doesn't support USDC or WBTC.  Both should work with the next `auction-keeper` release, which will include updated `pymaker` and  `exit_dai_on_shutdown`.

File: flip/wbtc-a/model-wbtc.sh (hunk 1 above)
Comment (godsflaw) [reply flip/wbtc-a/model-wbtc.sh:14]: yeah, many, but arbitrary precision and floating point math are the most notable benefits over bash.

**Review by godsflaw — APPROVED** [review]
LGTM

File: README.md [hunk 2]
```diff
@@ -19,34 +19,36 @@ https://docs.docker.com/compose/install/
 ### Setup flip keepers
 
 After following the setup procedure below, this keeper works out of the box under the following configuration:
-- Participates in up to 100 active ETH-A or BAT-A or USDC-A Flip auctions; it does not start new ones
+- Participates in up to 100 active ETH-A or BAT-A or USDC-A or WBTC-A Flip auctions; it does not start new ones
 - Begins scan at a prescribed auction id - we recommend starting at:
-  - `mainnet` - 4500
```
Comment (KentonPrescott) [inline README.md:24]: There have only been ~500 BAT Flip auctions, so I believe this line should be:
`mainnet - 4500 for ETH, 500 for BAT, and 0 for WBTC`

File: README.md [hunk 3]
```diff
@@ -19,34 +19,36 @@ https://docs.docker.com/compose/install/
...
-- Uses a pricing model that tracks the price of ETH | BAT | USDC via a public API and applies a `DISCOUNT` before participating
-- All logs from the keeper are saved and appended to a single file (`auction-keeper-flip-ETH-A.log` or `auction-keeper-flip-BAT-A.log` or `auction-keeper-flip-USDC-A.log`)
+- Uses a pricing model that tracks the price of ETH | BAT | USDC | WBTC via a public API and applies a `DISCOUNT` before participating
+- All logs from the keeper are saved and appended to a single file (`auction-keeper-flip-ETH-A.log` or `auction-keeper-flip-BAT-A.log` or `auction-keeper-flip-USDC-A.log` or `auction-keeper-flip-WBTC-A.log`)
 
-- Place unlocked keystore and password file for account address under `secrets` directory. The names of the keystore and password files will need to be updated in the `FLIP_ETH_A_ACCOUNT_KEY` | `FLIP_BAT_A_ACCOUNT_KEY` | `FLIP_USDC_A_ACCOUNT_KEY` in the env.
+- Place unlocked keystore and password file for account address under `secrets` directory. The names of the keystore and password files will need to be updated in the `FLIP_ETH_A_ACCOUNT_KEY` | `FLIP_BAT_A_ACCOUNT_KEY` | `FLIP_USDC_A_ACCOUNT_KEY` | `FLIP_WBTC_A_ACCOUNT_KEY` in the env.
```
Comment (KentonPrescott) [inline README.md:32]: Quick question: Why is it required to place unlocked keyword and password file under the `secrets` directory? Still learning about Docker ...

File: README.md [hunk 4]
```diff
@@ -90,6 +95,9 @@ flip-bat-a keeper
 flip-usdc-a keeper
```
Comment (KentonPrescott) [inline README.md:96]: Under the `Shutdown` section, I recommend mentioning that `./stop-keeper.sh ...` should be called in the same repository but in another shell. I made the rookie mistake of trying to execute the command in the shell where the keeper was running - I wouldn't be surprised if I'm not alone.

**Review by KentonPrescott — COMMENTED** [review]
Successfully ran `flip-wbtc-a`, but hit two errors:

1) With GasMode = 2, got this error: 
```
flip-wbtc-a    | auction-keeper: error: unrecognized arguments: --etherchain-gas-price
flip-wbtc-a exited with code 2
```

2) Error when calling `./cleanup.sh` because outdated image is still used by another local repo: 
```
./cleanup.sh
Removing flip-wbtc-a ... done
Removing network grandizzy-dockerized-auction-keeper_default
Error response from daemon: conflict: unable to remove repository reference "makerdao/auction-keeper" (must force) - container 949a3e14003a is using its referenced image 4b92fb48a28a
```
To fix it, I simply forced it: `docker-compose down && docker rmi -f makerdao/auction-keeper`

File: README.md (hunk 3 above)
Comment (grandizzy) [reply README.md:32]: `secrets` directory is mounted in Docker container under `/opt/keeper` path, see https://github.com/makerdao/dockerized-auction-keeper/blob/master/docker-compose.yml#L9 which is used in startup script to unlock keeper account https://github.com/makerdao/dockerized-auction-keeper/blob/master/env/environment.sh#L23

**Comment by grandizzy** [comment]
> Successfully ran `flip-wbtc-a`, but hit two errors:
> 
> 1. With GasMode = 2, got this error:
> 
> ```
> flip-wbtc-a    | auction-keeper: error: unrecognized arguments: --etherchain-gas-price
> flip-wbtc-a exited with code 2
> ```

Please try again now that you `./cleanup.sh` Issue was that you had a Docker image built with older auction keeper code that didn't have support for etherchain

**Review by KentonPrescott — APPROVED** [review]
LGTM

**Pull request merged**